### PR TITLE
[CDE-571] - IE 11 : Options Cancel and OK disable when adding HTML code

### PR DESCRIPTION
**Pull request by jvelasques**

### CODE REVIEW

**Comment by jvelasques** [comment]
@pamval please review

File: cde-core/resource/css/master.css [hunk 1]
```diff
@@ -1710,6 +1710,11 @@ div.popup {
   transition: 0.3s;
 }
 
+/* IE11 will behave strangely with prompts with position fixed inside PUC */
+_:-ms-fullscreen, :root div.popup {
+  position: absolute !important;
+}
```
Comment (pamval) [inline cde-core/resource/css/master.css]: does this really need the !important tag ?

Comment (jvelasques) [inline cde-core/resource/css/master.css]: unfortunately it does, its overwriting a css rule that has an !important tag

**Comment by jvelasques** [comment]
@pamval rebased with master, please review

**Comment by pamval** [comment]
cherry-pick to 5.x please

**Pull request merged**